### PR TITLE
Use wheels in non yelp env

### PR DESCRIPTION
**Pull request by jglukasik**
We've were getting some Travis build failures in compiling tornado. With this change we can use wheels for that when outside of the Yelp env.  This way we also don't need all the -dev packages

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -23,4 +23,4 @@ override_dh_auto_test:
 	true
 
 override_dh_virtualenv:
-	dh_virtualenv $EXTRA_PYPI --python=/usr/bin/python2.7 --preinstall no-manylinux1 --preinstall pip-custom-platform --pip-tool pip-custom-platform
+	dh_virtualenv $EXTRA_PYPI --python=/usr/bin/python2.7 --preinstall pip-custom-platform --pip-tool pip-custom-platform
```
Comment (asottile) [inline debian/rules]: you need `no-manylinux1`, manylinux wheels are not compatible with dh-virtualenv

this essentially defeats the purpose of this entire review :P

Comment (solarkennedy) [inline debian/rules]: Crap! why are manylinux wheels not compatible with dh-virtualenv?
You mean in some sort of technical sense, right? (not just yelp policy)

Comment (asottile) [inline debian/rules]: not just yelp policy: https://github.com/spotify/dh-virtualenv/issues/150

Comment (solarkennedy) [inline debian/rules]: Thank you for sharing the knowledge :)

**Pull request closed without merging**